internal/logging/defaults.go

```diff
@@ -72,6 +72,9 @@ func FilePath() string {
 }
 
 func FilePathFor(filename string) string {
+	if filepath.IsAbs(filename) {
```

Comment:
If the path is absolute it could still not be the correct log file path. We may want to look into the `GetLogFileName` of the `comm` package because it appears that it's not returning just the log file name but the entire path as well.

Reply:
Going up the chain it looks like the service is instantiated with a full path instead of just a name.